packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -7772,6 +7772,48 @@ QUnit.module('label integration', () => {
 });
 
 QUnit.module('accessibility', () => {
+    QUnit.test('Tags aria labels should match inputAttr aria-label mappings', function(assert) {
+        const productsData = [{
+            ID: 1,
+            Name: '1',
+        }, {
+            ID: 2,
+            Name: '2',
+        }];
+        const $tagBox = $('#tagBox').dxTagBox({
+            dataSource: productsData,
+            valueExpr: 'ID',
+            value: [productsData[0].ID, productsData[1].ID],
+            displayExpr: 'Name',
+            inputAttr: { 'aria-label': 'Name' }
+        }).dxTagBox('instance');
+        const inputAttrValue = $tagBox.option('inputAttr')['aria-label'];
+        const simpleTags = $tagBox.$element().find(`.${TAGBOX_TAG_CLASS}`);
+        const isTaggedCorrectly = simpleTags.toArray().every(tag => productsData.some(product => product[inputAttrValue] === $(tag).attr('aria-label')));
+
+        assert.strictEqual(isTaggedCorrectly, true, 'aria-labels are tagged correctly');
+    });
+
+    QUnit.test('Tag aria label should be the tags value when inputAttr is not defined', function(assert) {
+        const productsData = [{
+            ID: 1,
+            Name: '1',
+        }, {
+            ID: 2,
+            Name: '2',
+        }];
+        const $tagBox = $('#tagBox').dxTagBox({
+            dataSource: productsData,
+            valueExpr: 'ID',
+            value: [productsData[0].ID, productsData[1].ID],
+            displayExpr: 'Name',
+        }).dxTagBox('instance');
+        const simpleTags = $tagBox.$element().find(`.${TAGBOX_TAG_CLASS}`);
+        const isTaggedCorrectly = simpleTags.toArray().every((tag, index) => index + 1 === parseInt($(tag).attr('aria-label')));
```

Comment:
"You have only two items; let's check the aria-label attribute for each item in separate assertions without any loops

Reply:
[aea94cd](https://github.com/DevExpress/DevExtreme/pull/27157/commits/aea94cd1437771c23e23b2de7e712ee63a6db785)